src/UI/UIUtils.java

```diff
@@ -352,6 +355,25 @@ public enum SourceGames
       this.uiColorSpec = new COSpriteSpec(new Faction(this.name()), uiColorSpec.color);
       this.discriminator = discriminator;
     }
+
+    static // This block goes in here so the instances are constructed before it's executed. Null enums ain't fun.
+    {
+      CANON_FACTIONS = new COSpriteSpec[] { OS, BM, GE, YC, BH,
+          // Custom
+          RF, GS, BD, AB, JS, CI, PC, TG, PL, AR, WN, RT, CO, IDS, MISC,
+          // DoR
+          BW, NW, LA, };
+      // In order of CO name discriminator resolution
+      GAME_RESOLUTION_ORDER = new SourceGames[]
+        {
+            SourceGames.DEFEND_PEACE,
+            SourceGames.AW3,
+            SourceGames.AW2,
+            SourceGames.AW1,
+            SourceGames.AW4,
+            SourceGames.AWBW,
```

Comment:
3 2 1 4?

Reply:
That's what it was before.
The ordering is like:
- Canonical
- Newest Trilogy entry
- Older
- Oldest
- One of these things is not like the others.

It wouldn't hurt anything to put DoR first, I suppose.